Earthfile

```diff
@@ -9,3 +11,26 @@ proto:
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf build
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf generate
   SAVE ARTIFACT gen/proto/go/translate/v1 AS LOCAL pkg/server/translate/v1
+
+lint-go:
+  FROM golang:$go_version-alpine
+  RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v$golangci_lint_version
+  RUN apk add build-base
```

Comment:
Needed for golancgi-lint else error is thrown
https://github.com/golangci/golangci-lint/issues/3289